collect/collect.go

```diff
@@ -30,7 +30,7 @@ import (
 	"github.com/sirupsen/logrus"
 )
 
-var ErrWouldBlock = errors.New("not adding span, channel buffer is full")
+var ErrWouldBlock = errors.New("not adding span, channel buffer is full. Span will not be processed and will be lost.")
```

Comment:
I wonder if it would be easier to read this error message if the impact is stated first? Something like `Span lost. Not adding span, channel buffer is full`